src/app/zap-templates/zcl/trusted-root-certificates-cluster.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>General</domain>
+    <name>Trusted Root Certificates</name>
+    <code>0x003F</code> // Change to correct clusterID once it is decided in the spec #2396
```

Comment:
Please use a temporary code is this may clash with the next sequential valid identifier. Use "much beyond" current range for these not-yet-numbered clusters.

I am working with DATT in spec to see if we can move that single attribute and 2 commands to OpCreds cluster.

Reply:
Moving this to OpCreds will be much appreciated. This is a one-off cluster and making it complicated.